addons/stock_account/models/stock_move.py

```diff
@@ -269,7 +269,8 @@ def _action_done(self, cancel_backorder=False):
                 continue
             if svl.currency_id.is_zero(svl.value):
                 continue
-            svl.stock_move_id._account_entry_move(svl.quantity, svl.description, svl.id, svl.value)
+            description = svl.stock_move_id.reference + ' - ' + svl.product_id.name if svl.stock_move_id.reference else svl.product_id.name
```

Comment:
can you directly fix the description of the svl ? so the descriptions svl <-> am are aligned